hikari/api/rest.py

```diff
@@ -6852,7 +6852,7 @@ async def create_slash_command(
         default_member_permissions : typing.Union[hikari.undefined.UndefinedType, int, hikari.permissions.Permissions]
             Member permissions necessary to utilize this command by default.
 
-            If `0`, then it will be disabled by default. This excludes administrators
+            If `0`, then it will be available for all members. This excludes administrators
```

Comment:
I think this might need to be rewritten clearer, this confused me.

As with the other occurrences.